jumanji/environments/packing/bin_pack/space.py

```diff
@@ -134,6 +136,12 @@ def hyperplane(self, axis: str, direction: str) -> "Space":
         """
         inf_ = jnp.inf
         axis_direction = f"{axis}_{direction}"
+        if full_support:
+            supporting_item_space = self
```

Comment:
Is hyperplane only ever used for an item space or is it also used for other things? A comment to explain that this is a safe check to make because hyperplanes are only generated from item spaces?

Reply:
Done ✅